openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -810,10 +810,7 @@ The issued Credential SHOULD be cryptographically bound to the identifier of the
 
 Note: Claims in the Credential are about the subject of the Credential, which is often the End-User who possesses it.
 
-For Cryptographic Key Binding, the Client has the following options defined in (#credential-request) to provide Cryptographic Key Binding material for a requested Credential:
-
-1. Provide proof of control alongside key material.
-1. Provide only proof of control without the key material.
+For Cryptographic Key Binding, the Client has different options to provide Cryptographic Key Binding material for a requested Credential within a proof of a certain proof type. A proof type may provide the cryptographic public key(s) either with corresponding proof(s) of possession or with key attestations. Proof types are defined in (#proof-types).
```

Comment:
```suggestion
For Cryptographic Key Binding, the Client has different options to provide Cryptographic Key Binding material for a requested Credential within a proof of a certain proof type. A proof type may provide the cryptographic public key(s) either with corresponding proof(s) of possession of the private key or attestation. Proof types are defined in (#proof-types).
```

Reply:
"...or with key attestation" would be clearer to me

Reply:
Don't feel too strongly, happy with the original text, "proof of possession" alone is probably good enough without my addition.